### PR TITLE
Bugfix 2212/rewire report display

### PR DESCRIPTION
**Pull request by vhscom**
As discovered the previous page selector used in the Check-Ins was filtering out members without Check-Ins activity. Existing report logic is now replaced as expected per ACs in #2212 to report the expected PDL and team members.

Also updates our data load script to provide additional check-ins info to test reporting edge cases.

Finally, adds a keyboard listener to the Check-Ins report page for navigating quarters. This is especially helpful while making comparisons between quarters when the user's viewport has scrolled below the fold.

### CODE REVIEW

**Review by jackkeller — APPROVED** [review]
Approved

File: web-ui/src/components/reports-section/checkin-report/CheckinReport.jsx [hunk 1]
```diff
@@ -36,68 +39,51 @@ const CheckinsReport = ({ closed, pdl, planned, reportDate }) => {
...
-        planned
-      );
-      checkins = checkins.filter(checkin =>
-        isCheckinInProgress(checkin, startOfQuarter, endOfQuarter)
-      );
-      return checkins.length > 0;
-    };
-
-    const anyInProgress = members.some(isInProgress);
+    const anyInProgress = members.some(
```
Comment (mvolkmann) [inline web-ui/src/components/reports-section/checkin-report/CheckinReport.jsx:69]: I love use of the `some` and `every` `Array` methods!

**Review by mvolkmann — APPROVED** [review]
